components/kserve/kserve.go

```diff
@@ -54,6 +54,8 @@ type Kserve struct {
 	// This field is optional. If no default deployment mode is specified, Kserve will use Serverless mode.
 	// +kubebuilder:validation:Enum=Serverless;RawDeployment
 	DefaultDeploymentMode DefaultDeploymentMode `json:"defaultDeploymentMode,omitempty"`
+	// +kubebuilder:validation:Type:bool
+	DisableIngressCreation bool `json:"disableIngressCreation,omitempty"`
```

Comment:
As per the [Open Shift API guidelines](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#do-not-use-boolean-fields), boolean should be avoided if not strictly necessary.

To follow other conventions, eventually this could changed to be an enum, like `ingress: Managed|Unmanaged ` which defaults to `Managed` which to me seems to be more clear and avoid using [negative names](https://dart.dev/effective-dart/design#prefer-the-positive-name-for-a-boolean-property-or-variable)

Reply:
Changed it to be a enum